tests/components/dynalite/test_init.py

```diff
@@ -7,11 +7,13 @@
 from voluptuous import MultipleInvalid
 
 import homeassistant.components.dynalite.const as dynalite
-from homeassistant.const import CONF_HOST, CONF_NAME, CONF_PORT, CONF_ROOM
+from homeassistant.const import CONF_DEFAULT, CONF_HOST, CONF_NAME, CONF_PORT, CONF_ROOM
 from homeassistant.setup import async_setup_component
 
 from tests.common import MockConfigEntry
 
+dynalite.CONF_DEFAULT = CONF_DEFAULT
```

Comment:
This is hacky. Please adjust the tests to use CONF_DEFAULT directly.

Reply:
You're right. Done